src/commands/remember/remember-count.js

```diff
@@ -1,5 +1,42 @@
 const {ApplicationCommandOptionType} = require('discord.js');
-const {saveNumberMessages} = require("../../utils/rememberMessages");
+const {parseMessage, addMessages} = require("../../utils/rememberMessages");
+const {defaultExcludeBotMessages} = require('../../../config.json');
+const {getNumberMessages} = require('../../utils/apiCalls');
+
+const getMessagesAndReturnId = async(messagesToSave, channel, num, excludeBotMessages, startId) =>{
+    let messageObjArray;
+    if(startId){
+         //get the first 100 messages at a specifc point
+         messageObjArray = await getNumberMessages(channel, num, startId)
+    }
+    else{
+        //get the num messages starting with the latest one
+        messageObjArray = await getNumberMessages(channel, num)
+    }
```

Comment:
```suggestion
    messageObjArray = await getNumberMessages(channel, num, startId);
```
This should be the same thing; if there's no startId, it's `undefined` anyway.